### PR TITLE
update so files are created in the correct locations

### PR DESCRIPTION
**Pull request by KarenWGard**

### CODE REVIEW

File: rct229/ruletest_engine/ruletest_jsons/scripts/excel_to_test_json.py [hunk 1]
```diff
@@ -328,7 +329,7 @@ def create_dictionary_from_excel(spreadsheet_name, sheet_name):
     return json_dict
 
 
-def create_test_json_from_excel(spreadsheet_name, sheet_name, json_name):
```
Comment (weilixu) [inline rct229/ruletest_engine/ruletest_jsons/scripts/excel_to_test_json.py:331]: If we move all the master jsons to the ruleset, we will need to update the function in disaggregate_master_ruletest_json to take the same path.

**Comment by KarenWGard** [comment]
Juan - this sounds good to me.

Thanks!

**Comment by weilixu** [comment]
@jugonzal07 @KarenWGard do we still need this PR?

**Comment by KarenWGard** [comment]
no - I think it duplicated a fix that Juan did.

**Pull request closed without merging**